editor/RubyScanner.cpp

```diff
@@ -140,7 +140,7 @@ bool Scanner::didBlockStart()
 
 bool Scanner::didBlockEnd()
 {
-    static const QRegExp regex(QLatin1String("24_"));
+    static const QRegExp regex(QLatin1String("(24|29)_"));
```

Comment:
Hmmm... sorry, some time I don't touch this code... I'll check it in detail later to check if it doesn't screw up with the indentation when if and parenthesis are in the same line.